client/ayon_core/pipeline/context_tools.py

```diff
@@ -579,3 +579,14 @@ def get_process_id():
     if _process_id is None:
         _process_id = str(uuid.uuid4())
     return _process_id
+
+
+def version_up_workfile():
+    """Function to increment and save workfile
+    """
+    host = registered_host()
+    current_file = host.get_current_workfile()
+    if not current_file:
+        return None
```

Comment:
Should this at least log a warning?

Another approach could be to use maya menu's `postMenuCommand` argument to trigger a callback whenever the AYON menu opens, and if no current scene name is set that _then_ we disable the `Version Up Workfile` button.

Reply:
```suggestion
        return
```